### PR TITLE
Add packet size considerations

### LINKED ISSUE #33: Provide advice about initial MSS of QUIC connections to proxies
Rather that just using 1200 bytes, QUIC connections to proxies should add extra bytes to account for proxied QUIC connections that will themselves send initial packets of 1200 bytes that will need to fit in DATAGRAM frames.

### PR DESCRIPTION
**Pull request by tfpauly**
Closes #33

### CODE REVIEW

File: draft-pauly-masque-quic-proxy.md [hunk 1]
```diff
@@ -576,6 +576,22 @@ to the proxy sending 2xx (Successful) responses to clients.
...
+# Packet Size Considerations
+
+Since Initial QUIC packets must be at least 1200 bytes in length, the DATAGRAM
+frames that are used for a QUIC-aware CONNECT-UDP proxy MUST be able to carry
+at least 1200 bytes.
+
+Additionally, clients that connect to a proxy for purpose of proxying QUIC
+SHOULD start their connection with a larger packet size than 1200 bytes, to
+account for the overhead of tunnelling an Initial QUIC packet within a
+DATAGRAM frame. If the client does not being with a larger packet size, it
```
Comment (DavidSchinazi) [inline draft-pauly-masque-quic-proxy.md:588]: This doesn't parse?

Comment (tfpauly) [reply draft-pauly-masque-quic-proxy.md:588]:
```suggestion
DATAGRAM frame. If the client does not begin with a larger packet size than 1200 bytes, it
```

File: draft-pauly-masque-quic-proxy.md [hunk 2]
```diff
@@ -576,6 +576,22 @@ to the proxy sending 2xx (Successful) responses to clients.
...
+# Packet Size Considerations
+
+Since Initial QUIC packets must be at least 1200 bytes in length, the DATAGRAM
+frames that are used for a QUIC-aware CONNECT-UDP proxy MUST be able to carry
+at least 1200 bytes.
+
+Additionally, clients that connect to a proxy for purpose of proxying QUIC
+SHOULD start their connection with a larger packet size than 1200 bytes, to
+account for the overhead of tunnelling an Initial QUIC packet within a
+DATAGRAM frame. If the client does not begin with a larger packet size than 1200 bytes, it
```
Comment (DavidSchinazi) [inline draft-pauly-masque-quic-proxy.md:588]: you'll probably want to reflow this to match the line length limit

**Review by DavidSchinazi — APPROVED** [review]

File: draft-pauly-masque-quic-proxy.md [hunk 3]
```diff
@@ -576,6 +576,22 @@ to the proxy sending 2xx (Successful) responses to clients.
...
+will need to perform Path MTU (Maximum Transmission Unit) discovery to
+discover a larger path size prior to sending any tunnelled Initial QUIC packets.
+
+Once a proxied QUIC connections moves into forwarded mode, the client SHOULD
```
Comment (chris-wood) [inline draft-pauly-masque-quic-proxy.md:592]: (I can't suggest)

nit: s/connections/connection

**Pull request merged**